pkg/karmadactl/cmdinit/utils/util_test.go

```diff
@@ -24,3 +25,24 @@ func TestDownloadFile(t *testing.T) {
 		panic(err)
 	}
 }
+
+func TestListFiles(t *testing.T) {
+	tests := []struct {
+		name string
+		path string
+		want []string
+	}{
+		{
+			name: "files",
+			path: "../options",
+			want: []string{"../options/global.go"},
```

Comment:
The result of this test case will depend on the file in the `../options` path.

Reply:
what good advice do you have @XiShanYongYe-Chang

Reply:
I don't have any ideas yet.